packages/accordions/src/elements/stepper/components/Content.tsx

```diff
@@ -11,6 +11,9 @@ import debounce from 'lodash.debounce';
 import { StyledContent, StyledInnerContent } from '../../../styled';
 import { useStepContext, useStepperContext } from '../../../utils';
 
+/**
+ * @extends HTMLDivElement
```

Comment:
Should be `HTMLAttributes<HTMLDivElement>`

Reply:
Fixed.

Reply:
extends have been removed for now.